firestore/main/index.js

```diff
@@ -63,17 +63,17 @@ async function demoInitialize(db) {
 // ============================================================================
 
 async function quickstartAddData(db) {
-  // [START add_lovelace]
+  // [START firestore_setup_dataset_pt1]
```

Comment:
@laljikanjareeya we need to keep both tags for now. I know this is a bit verbose, but ensures we don't lose any if we try to generate docs between this change and updates to our docsite. 

Could you make these read like

```
// [START add_lovelace]
// [START firestore_setup_dataset_pt1]


/* SAMPLE CODE */

// [END firestore_setup_dataset_pt1]
// [END add_lovelace]

Reply:
Done!!